app.py

```diff
@@ -19,24 +19,29 @@
 import re
 import traceback
 from flask_commonmark import Commonmark
+from babel import Locale
 from werkzeug.urls import url_encode
 from config import config
-from website.auth import current_user, login_user_from_token_cookie, requires_login, is_admin, is_teacher, update_is_teacher
-from utils import timems, load_yaml_rt, dump_yaml_rt, version, is_debug_mode
-import utils
-import textwrap
-
-# app.py
 from flask import Flask, request, jsonify, session, abort, g, redirect, Response, make_response, Markup
 from flask_helpers import render_template
 from flask_compress import Compress
 from flask_babel import Babel, refresh
 from flask_babel import gettext
 
+# Setting up Flask and babel (web and translations)
+app = Flask(__name__, static_url_path='')
+app.url_map.strict_slashes = False # Ignore trailing slashes in URLs
+babel = Babel(app)
+
 # Hedy-specific modules
 import hedy_content
+hedy_content.fill_all_languages(babel)
 import hedyweb
-from hedy_content import ALL_LANGUAGES, FALL_BACK_ADVENTURE, ALL_KEYWORD_LANGUAGES
+from hedy_content import ALL_LANGUAGES, ALL_KEYWORD_LANGUAGES
```

Comment:
Is there a reason this part of the header is moved down (I thin kit is good style to have all imports at the top)

Reply:
Yes, the line above it: hedy_content.fill_all_languages(babel). Defined and filled those dictionaries. This didn't work for Timon, so I had to declare the dict empty. this code could probably be lowered. The import is an empty dict then, but it's fille later.

Reply:
I didn't work for our tests either, Python is a bit ambiguous on importing a variabele that isn't there (yet). We might be able to solve this by creating a dedicated Translations class that has an object that keeps track of all our translations, but creating an empty dict first seems to work fine as well.